File: Codechef/Tv_Disc/Tv_Disc.cpp
#include <iostream>
using namespace std;
int main()
{
    int T;
    cin >> T;
    while (T--)
    {
        int A, B, C, D;
        cin >> A >> B >> C >> D;
        A -= C;
        B -= D;
        if (A < B)
        {
            cout << "First" << endl;
        }
        else if (B < A)
        {
            cout << "Second" << endl;
        }
        else
        {
            cout << "Any" << endl;
        }
    }
    return 0;
}

File: Codechef/Tv_Disc/Tv_Disc.py
T = int(input())
while T:
    T -= 1
    A, B, C, D = map(int, input().split())
    A -= C
    B -= D
    if A < B:
        print("First")
    elif B < A:
        print("Second")
    else:
        print("Any")
